.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "mini-gdbstub"]
 	path = src/mini-gdbstub
 	url = https://github.com/RinHizakura/mini-gdbstub
+[submodule "SoftFloat-3e"]
+	path = src/SoftFloat-3e
```

Comment:
Simply name it `src/softfloat` as QEMU does.
See https://github.com/qemu/qemu/blob/master/fpu/softfloat.c